src/components/LayerSwitcher/AddCustomLayer.tsx

```diff
@@ -218,6 +218,28 @@ const Details: React.FC<Detailsprops> = ({ layer, onChange, onValidation }) => {
   );
 };
 
+const humanJoin = (arr: string[]) => {
+  if (arr.length === 0) return '';
+  if (arr.length === 1) return arr[0];
+  if (arr.length === 2) return arr.join(' and ');
+
+  return arr.slice(0, -1).join(', ') + ' and ' + arr[arr.length - 1];
+};
+
+const MissingParts = ({ x, y, z }) => {
+  const missingParts = [
+    ...(!x ? ['{x}'] : []),
+    ...(!y ? ['{y}'] : []),
+    ...(!z ? ['{z}'] : []),
+  ];
+
+  if (!missingParts.length) {
+    return null;
+  }
+
+  return <p>Insert {humanJoin(missingParts)}</p>;
```

Comment:
According to [docs](https://maplibre.org/maplibre-gl-js/docs/API/classes/RasterTileSource/), there are two options:

- use `{x}`, `{y}`, `{z}`
- or `{bbox-epsg-3857}`

I think it would be ok, just to add a helper text like "Please insert {z},{x},{y} tokens or {bbox-epsg-3857}", it would be easier to translate it. But your interactive evaluation is pretty cool :-)